nexus/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM image-registry.openshift-image-registry.svc:5000/openshift/nexus-repository-manager
```

Comment:
Can you please verify that this works on both OCP 3 & 4?
If this only works on 4 then I think we need to be explicit about that in the README.